doc/deployment_guide.md

```diff
@@ -0,0 +1,6080 @@
+
+![Figure 2 OvS Architecture](media/943bbeaf066c3acddca7d92e304680de.png)
+
+As can be seen in the Figure 2, OVS kernel module acts as a cache for the user space. And just like a cache, its performance decreases as the number of rules increase in the user space.
+
+#### DPDK
+
+The Data Plane Development Kit (DPDK) is a set of data plane libraries and network interface controller drivers for fast packet processing from Intel. DPDK runs inside user space, and gives applications direct access to raw traffic from NIC, completely bypassing kernel and kernel IP Stack. It further utilizes poll mode drivers (PMDs) and Hugepages to increase network performance.
+
+
+#### OVS-DPDK
+
+With the help of DPDK, all the OVS processes are moved to user space. Since all the paths are in single space, it removes the bottleneck created by continuous switching between two spaces.
+
+
+#### OVS-DPDK in Dell EMC Ready Architecture for Red Hat OpenStack Platform v13
+
+OVS-DPDK is enabled as a part of the automated RHOSP deployment in Dell EMC Ready Architecture for Red Hat OpenStack Platform Version 13.0. There is no post deployment enablement support in this release. If enabled, OVS-DPDK must be so on all compute nodes.
+
+> Note: OVS-DPDK deployment is enabled only on Dell Compute role.
+ 
+
+OVS-DPDK requires an extra network bond; the already existing bonds (bond0 and bond1) carry storage network, OpenStack tunnelled tenant network and internal API network. The OVS-DPDK bond will carry OpenStack VLAN tenant networks.
+
+#### Before You Begin
+
+In this guide, it is assumed that the user has complete knowledge about the Dell EMC Ready Architecture for Red Hat OpenStack Platform Version 13. This includes:
+1.	Knowledge about different nodes in Dell EMC Ready Architecture for Red Hat OpenStack Platform version 13, like SAH, Director, Controller, Compute and Ceph-storage as explained in  Dell_EMC_Red_Hat_Ready_Architecture_Guide_v13.0.pdf
+2.	Hardware configurations including switch configurations as explained in Dell_EMC_Red_Hat_Ready_Architecture_Guide_v13.0.pdf and Dell_EMC_Red_Hat_Ready_Architecture_Cumulus_Switch_Configurations_v13.0.pdf
+3.	Automation scripts, settings, and properties files required for deployment are open sourced and available in git hub https://github.com/dsp-jetpack/JetPack
+

```

Comment:
update to https://github.com/dsp-jetpack/JetPack/tree/JS-13.0

Reply:
updated